Airbrake_Ideas_Test.py

```diff
@@ -0,0 +1,205 @@
+(1263.37639001725,-66.8356625683275),
+(1249.85628244493,-68.3621351366883),
+(1236.03285333105,-69.8688478276226),
+(1221.91007209790,-71.3556293986152),
+(1207.49194049068,-72.8223278134795),
+(1192.78248876700,-74.2688099295736),
+(1177.78577195182,-75.6949611574194),
+(1162.50586616296,-77.1006850947060),
+(1146.94686501227,-78.4859031366639),
+(1131.11287608690,-79.8505540647998),
+(1115.00801751483,-81.1945936159693),
+(1098.63641461839,-82.5179940337477),
+(1082.00219665909,-83.8207436040302),
+(1065.10949367678,-85.1028461767609),
+(1047.96243342569,-86.3643206756471),
+(1030.56513840957,-87.6052005976695),
+(1012.92172301777,-88.8255335041442),
+(995.036290763900,-90.0253805050372),
+(976.912931628059,-91.2048157381662),
+(958.555719503678,-92.3639258448634),
+(939.968709749422,-93.5028094435995),
+(921.155936846469,-94.6215766030010),
+(902.121412161117,-95.7203483156169),
+(882.869121812474,-96.7992559737187),
+(863.403024644665,-97.8584408483388),
+(843.727050302825,-98.8980535726780),
+(823.845097411878,-99.9182536309366),
+(803.761031856933,-100.919208853545),
+(783.478685163909,-101.901094919699),
+(763.001852978859,-102.864094868023),
+(742.334293644286,-103.808398616120),
+(721.479726870593,-104.734202489694),
+(700.441832500716,-105.641708761848),
+(679.224249365818,-106.531125203123),
+(657.830574229867,-107.402664642738),
+(636.264360820815,-108.256544541466),
+(614.529118945998,-109.092986576497),
+(592.628313689349,-109.912216238593),
+(570.565364687920,-110.714462441778),
+(548.343645485206,-111.499957145759),
+(525.966482958691,-112.268934991219),
+(503.437156819044,-113.021632948091),
+(480.758899178358,-113.758289976857),
+(457.934894184818,-114.479146702889),
+(434.968277721211,-115.184445103815),
+(411.862137164651,-115.874428209841),
+(388.619511204959,-116.549339816949),
+(365.243389719127,-117.209424212837),
+(341.736713699330,-117.854925915463),
+(318.102375231976,-118.486089424017),
+(294.343217525349,-119.103158982122),
+(270.462034983395,-119.706378353048),
+(246.461573323289,-120.295990606712),
+(222.344529734441,-120.872237918201),
+(198.113553076671,-121.435361377571),
+(173.771244115314,-121.985600810627),
+(149.320155791100,-122.523194610416),
+(124.762793522686,-123.048379579122),
+(100.101615539794,-123.561390780074),
+(75.3390332449616,-124.062461399555),
+(50.4774116019766,-124.551822618094),
+]
+# calcs = Calculations()
+pred_alt = 0
+
+def predicted_alt(alt,velocity): #Maybe read in density too?
+        m=33.0693 #lbs 
+        Cd=0.42 #CHANGE for each rocket!! 
+        A= 0.02411 # ft^2 TODO Brenner check that this seems right
+        rho=0.069607176 #lbs/ft^3 at 2000ft 
+        g=32.16789 #ft/s^2 
+        Xc=(m/(rho*Cd*A)*math.log((m*g+0.5*rho*Cd*A*velocity**2)/(m*g)))+alt
+        print(Xc)
+        return Xc
+
+def test_calculations():
+    # Runs each velocity/altitude combo
+    for alt, vel, in data:
+        pred_alt = predicted_alt(alt*3.28, vel*3.28)
```

Comment:
Do we need to add a "global" identifier to pred_alt to make sure it uses the global pred_alt and not create a local one?